SimPPS/PPSPixelDigiProducer/interface/RPixSignalPoint.h

```diff
@@ -0,0 +1,32 @@
+#ifndef SimPPS_PPSPixelDigiProducer_RPix_SignalPoint_H
+#define SimPPS_PPSPixelDigiProducer_RPix_SignalPoint_H
+
+#include "DataFormats/GeometryVector/interface/LocalPoint.h"
+#include "DataFormats/GeometryVector/interface/LocalVector.h"
+
+/**
+ * An elementar charge point, with position, sigma from diffusion and tof.
+ */
+class RPixSignalPoint {
+public:
+  RPixSignalPoint() : pos_(0, 0), sigma_(0), charge_(0){};
+
+  RPixSignalPoint(double x, double y, double s, double charge) : pos_(x, y), sigma_(s), charge_(charge){};
+
+  inline LocalPoint Position() const { return pos_; }
+  inline double X() const { return pos_.x(); }
```

Comment:
This code comes from old TOTEM code. For sure I can make such a change, but it would delay the merge even further. I would kindly ask to keep this change as a change for after the merge, would it be possible, please?